components/brave_wallet_ui/common/api/hardware_keyrings.ts

```diff
@@ -48,10 +49,10 @@ export function getHardwareKeyring (
   type: HardwareVendor,
   coin: BraveWallet.CoinType = BraveWallet.CoinType.ETH,
   onAuthorized?: () => void
-): LedgerBridgeKeyring | HWInterfaces.TrezorKeyring | FilecoinLedgerKeyring | SolanaLedgerBridgeKeyring {
+): EthereumLedgerBridgeKeyring | HWInterfaces.TrezorKeyring | FilecoinLedgerKeyring | SolanaLedgerBridgeKeyring {
```

Comment:
Hm, I'm not sure what the thinking here is either.. I'd imagine we'd want to use the interface type for functions like `getHardwareKeyring` so that we can swap implementations, though I doubt there is significant benefit there in practice.

In that case though, then I'd expect we'd use the interface types for the `HardwareKeyring` declaration above.  So
```export type HardwareKeyring = EthereumLedgerBridgeKeyring | TrezorBridgeKeyring | SolanaLedgerBridgeKeyring```
would be come
```export type HardwareKeyring = HWInterfaces.LedgerEthereumKeyring | HWInterfaces.TrezorKeyring | HWInterfaces.LedgerFilecoinKeyring | HWInterfaces.LedgerSolanaKeyring```